index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>

```

Comment:
- kindly add viewport meta tag in the head

Reply:
- [ ] [REQUIRED] Please, make sure that you mention the reason for the requested changes so that the student can learn for his mistakes